src/helix-ui/styles/components/radios.less

```diff
@@ -0,0 +1,262 @@
+@import (reference) 'mixins';
+@import (reference) 'vars';
+
+// base styles
```

Comment:
pls remove lines 4-10, guessing these were used for guidance of css placement but not needed anymore?

Reply:
comments removed